cmd/ddev/cmd/addon-list.go

```diff
@@ -25,13 +25,18 @@ var AddonListCmd = &cobra.Command{
 	Example: `ddev add-on list
```

Comment:
To avoid this: `ddev addon list ddev/ddev-redis --all`

```suggestion
	Args:  cobra.NoArgs,
	Example: `ddev add-on list
```

Reply:
Ive been doing a lot of work with symfony/console lately where no args is the default validation 😅
Good catch, I'll update this tomorrow when I get a chance.